### PR TITLE
Added detail about GCCH and DoD meeting limits to limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by dariomws**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7204.

### CODE REVIEW

**Comment by dariomws** [comment]
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/7217.

@cichur could you kindly help to confirm this information? It seems that there is a lot of confusion about this sentence. Also, we should confirm if the 20,000 attendees limit will be available through June 30, 2021 or end of the year.

Thanks!

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -126,7 +126,7 @@ Channel names also can't start with an underscore (_) or period (.), or end with
 
 |Feature     | Maximum limit |
 |------------|---------------|
-|Number of people in a meeting (can chat and call in)  | 1000, includes GCC but not yet for GCCH or DoD. **View-only** allows for up to 20,000 listen-only participants to join a meeting in which the organizer has a license for E3/E5/A3/A5 SKU, as well as, Government (GCC, GCC High, DoD). Learn more about the [View-only experience](view-only-meeting-experience.md).|
+|Number of people in a meeting (can chat and call in)  | 1000, includes GCC but not yet for GCCH, DoD or A1 (300). **View-only** allows for up to 20,000 listen-only participants to join a meeting in which the organizer has a license for E3/E5/A3/A5 SKU, as well as, Government (GCC, GCC High, DoD). Learn more about the [View-only experience](view-only-meeting-experience.md).|
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:129]:
```suggestion
|Number of people in a meeting (can chat and call in)  | 1000, includes GCC but not yet for GCCH, DoD, or A1 (300). **View-only** allows for up to 20,000 listen-only participants to join a meeting in which the organizer has a license for E3/E5/A3/A5 SKU, as well as, Government (GCC, GCC High, DoD). Learn more about the [View-only experience](view-only-meeting-experience.md).|
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - minor edit.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**